__tests__/app.test.js

```diff
@@ -41,6 +41,50 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  describe("GET /api/articles/:article_id", () => {
+    test("GET 200: Responds with the required article object", () => {
+      return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.article.article_id).toBe(3);
+          expect(response.body.article.title).toBe(
+            "Eight pug gifs that remind me of mitch"
```

Comment:
apart from the article id I would have the rest as type of string etc :) if any of these were to be changed it would break this test

Reply:
Also done! Thank you for all the feedback!